common/src/main/java/io/netty/util/internal/UnpaddedInternalThreadLocalMap.java

```diff
@@ -32,7 +32,7 @@
  */
 class UnpaddedInternalThreadLocalMap {
 
-    static ThreadLocal<InternalThreadLocalMap> slowThreadLocalMap;
+    static final ThreadLocal<InternalThreadLocalMap> slowThreadLocalMap = new ThreadLocal<InternalThreadLocalMap>();
```

Comment:
@buchgr I didn't do that because it breaks `getIfSet`. @trustin FYI, when no thread local is set, slowThreadLocalMap.get will return null as its initialValue is not set.